doc/holochain_101/src/scenario_testing_setup.md

```diff
@@ -17,11 +17,11 @@ ___
 
 **Type** `object`
 
-**Description** *conductorOptions.debugLog* `boolean` Which logger type to use. There are two options:
-- debugLog = true: Use the "debug" logger. This one has nicer, colorful output.
-- debugLog = false: Use the "simple" logger. This one has less interesting output, but can be silenced with the env variable `HC_SIMPLE_LOGGER_MUTE=1`
+**Description** *conductorOptions.debugLog* `boolean` Which logging to use. There are two options:
+- debugLog = true: Use the "debug" logger. This one has full debug logging and nice, colorful output.
+- debugLog = false: Doesn't output any logs.
 
-**Default** `{ debugLog: true }`
```

Comment:
This whole thing could be simplified, people know that the two options for a `boolean` are `true` and `false`

**Description** *conductorOptions.debugLog* `boolean` Enables debug logging. The logger produces nice, colorful output of the internal workings of Holochain.
**Default** `{ debugLog: false }`